ocaml/xapi/xapi_clustering.ml

```diff
@@ -562,9 +562,18 @@ module Watcher = struct
         | None ->
             ()
       )
-      | Error (InternalError "UPDATES.Timeout") ->
-          (* UPDATES.get timed out, this is normal, now retry *)
-          ()
+      | Error (InternalError "UPDATES.Timeout") -> (
+        (* UPDATES.get timed out, this is normal.  *)
+        match find_cluster_host ~__context ~host with
+        | Some ch ->
+            let cluster = Db.Cluster_host.get_cluster ~__context ~self:ch in
+            (* CA-395789: We send a query to xapi-clusterd to fetch the latest state
+               anyway in case there is a race and the previous update did not give the
+               most up-to-date information *)
+            on_corosync_update ~__context ~cluster ["routine updates"]
```

Comment:
The log in `on_corosync_update` will print:
`Received 1 updates from corosync_notifyd`
But in this case, there is no updates found, how about call `on_corosync_update` with an empty updates list?

Reply:
I have adjusted the debug message according to the arguments